RELEASING

```diff
@@ -0,0 +1,25 @@
+# Scala.js Release Process
+
+1. Clean up bugs:
+    - Make sure all bugs assigned to the milestone are closed.
+    - Make sure all fixed bugs are assigned to the milestone ([this query][1]
+      should not return anything).
```

Comment:
I think we need more labels. For example:
* `can't reproduce` for #2223
* `separate repo` for #2168, #2039